asciidoc/modules/con_mg_miscellaneous-changes-in-vertx.adoc

```diff
@@ -0,0 +1,60 @@
+[id="miscellaneous-changes-in-vertx_{context}"]
+= Miscellaneous changes in {VertX}
+
+The following section describes miscellaneous changes in {VertX} {4}.
+
+== Removed the `Starter` class
+
+The `Starter` class has been removed. Use the `Launcher` class instead to start your {VertX} applications without the `main()` method.
+
+== Isolated deployment for Java 8
+
+{VertX} {4} supports Java 11. This Java version does not support isolated class loading.
+In {VertX} {4}, isolated class loading will be supported for Java 8. However, the results may not be as expected.
+
+== Removed hook methods from context
+
+The methods `Context.addCloseHook()` and `Context.removeCloseHook()` methods have been removed from the `Context` class.
```

Comment:
@vietj Does this note read ok? We do not need to explicitly mention that they have been moved to internal interface "InternalContext".

Original source:
https://github.com/vert-x3/wiki/wiki/4.0.0-Deprecations-and-breaking-changes#%EF%B8%8F-context-close-hook-made-internal

Reply:
I think it is good to tell where those are methods are in case a user need to use them because his application was relying on it.

Reply:
@vietj Added the following sentence:
== Removed hook methods from context

The methods `Context.addCloseHook()` and `Context.removeCloseHook()` methods have been removed from the `Context` class. **These classes have been moved to the internal interface `InternalContext`.**

Reply:
this is good. I think you can rename also _Removed hook methods from context_ to _Removed hook methods from Vert.x context_ because _context_ is a very overloaded word and it's good to add precision here.

Reply:
Done @vietj